specification.markdown

```diff
@@ -143,17 +143,17 @@ VINT_WIDTH  | VINT_MARKER  | VINT_DATA      | Element ID Status
 0           | 1            | 00000000111111 | Invalid: A shorter VINT_DATA encoding is available.
 |           | 1            |        1111111 | Invalid: VINT_DATA MUST NOT be set to all 1
 0           | 1            | 00000001111111 | Valid
-Table: Examples of valid and invalid VINTs {#tableVintValidity}
+Table: Examples of valid and invalid Element IDs {#tableElementIDValidity}
 
-The range and count of VINT_DATA values is determined by the octet length of the VINT. Examples of this are provided in [@tableVintRanges].
+The range and count of possible Element IDs are determined its the octet length. Examples of this are provided in [@tableElementIDRanges].
```

Comment:
Is the 1st sentence correct in English?

Reply:
"by their octet length" is probably more correcter.